src/clover/simulation/clinic.py

```diff
@@ -0,0 +1,335 @@
+
+    return t1 + t2 + t3 + t4
+
+
+def internal_load_people_1(building: Clinic):
+    """
+    Computes the internal heat load of the nurses in kWh for a building.
+
+    """
+    # load_p = np.where((time_people>=11)&(time_people<18), building.people_1 * 1 * building.heat_loss_people_1 / 1000., 0.)
+    people_at_times = [
+        [
+            [0, building.people_1 * building.heat_loss_people_1][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_people_1, building.end_time_people_1)
+    ]
+    return people_at_times
+
+
+def internal_load_people_2(building: Clinic):
+    """
+    Computes the internal heat load of the patients and accompainers in kWh for a building.
+
+    """
+    people_at_times = [
+        [
+            [0, building.people_2 * building.heat_loss_people_2][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_people_2, building.end_time_people_2)
+    ]
+    return people_at_times
+
+
+def calculate_internal_load_people(building: Clinic):
+    """
+    Computes the internal heat load of the total people in kWh for a building.
+
+    """
+    p1 = internal_load_people_1(MY_CLINIC)
+    p2 = internal_load_people_2(MY_CLINIC)
+
+    print(p1[i] + p2[i] for i in range(len(p1[0])))
+
+
+calculate_internal_load_people(MY_CLINIC)
+
+
+def internal_load_lighting(building: Clinic):
+    """
+    Computes the internal heat load of the lamps in kWh for a building.
+
+    """
+    lamps_at_times = [
+        [
+            [0, building.lamps_internal * building.wattage_lamps][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_lamps, building.end_time_lamps)
+    ]
+    return lamps_at_times
+
+
+def equipment_load(building: Clinic):
+    """
+    Computes the internal heat load of the fridge in kWh for a building.
+
+    """
+
+    return building.fridge_wattage * building.time_fridge / 1000
+
+
+def infiltration_load(building: Clinic, temperature):
+    """
+    Computes the infiltration load in kWh for a building.
+
+    """
+    infiltration_at_times = [
+        [
+            [
+                0,
+                building.changes
+                * building.volume_cold_store
+                * building.energy_new_air
+                * (temperature - building.inside_ideal_temperature),
+            ][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(
+            building.start_time_infiltration, building.end_time_infiltration
+        )
+    ]
+    return infiltration_at_times
+
+
+# infiltration_load(MY_CLINIC)
+
+
+def calculate_cooling_load(building: Clinic, temperature):
+    """
+    Computes the total cooling load in kWh for a building.
+
+    """
+
+    q1 = calculate_transmission_load(MY_CLINIC, temperature)
+    q2 = calculate_internal_load_people(
```

Comment:
Yes, this is the most problematic part in my opinion.. I don't really understand why it wants a parameter that is not in the function and what value or tuple should I add. In the calculate internal load people if I include the calculation in a return instead of print it doesn't really work so I'm not sure on what to do..